src/Mesh/babylon.abstractMesh.ts

```diff
@@ -763,7 +763,7 @@
             var yaw = -Math.atan2(dv.z, dv.x) - Math.PI / 2;
             var len = Math.sqrt(dv.x * dv.x + dv.z * dv.z);
             var pitch = Math.atan2(dv.y, len);
-            Quaternion.RotationYawPitchRollToRef(yaw + yawCor, pitch + pitchCor, rollCor, this.rotationQuaternion);
+            this.rotationQuaternion = Quaternion.RotationYawPitchRollToRef(yaw + yawCor, pitch + pitchCor, rollCor, this.rotationQuaternion || new Quaternion());

```

Comment:
Quaternion.RotationYawPitchRollToRef doesn't return the quaternion. It only copies to the reference quaternion.
better would be to set the quaternion of the parent, if not set before. 
for example:
```javascript
this.rotationQuaternion = this.rotationQuaternion || new Quaternion() 
```